deepset_cloud_sdk/__init__.py

```diff
@@ -14,6 +14,7 @@
     PipelineConfig,
     PipelineInputs,
     PipelineOutputs,
+    PipelineOutputType,
```

Comment:
forgot to add this in https://github.com/deepset-ai/deepset-cloud-sdk/commit/d370b982e0e99196ad6bb9fcf4c65f12a9875101